### PR TITLE
[T2A4][F12-A3]Gabriel Sim

### PR DESCRIPTION
**Pull request by gabrielsim**

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -482,10 +482,10 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      * @param keywords for searching
      * @return list of persons in full model with name containing some of the keywords
      */
-    private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
+    private static ArrayList<String[]> getPersonsWithNameContainingAnyKeywordCaseInsensitive(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
+            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person).toLowerCase()));
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:488]: please update the tests to reflect this change?
also ensure that this change doesn't affect the way data is stored internally.

**Review by okkhoy — COMMENTED** [review]
please note that when you update the functionality of the product you need to update 2 more things.
1. the documentation to reflect the change in functionality
2. tests to ensure you haven't introduced any bugs.

**Comment by okkhoy** [comment]
@gabrielsim1993 please note that the commit message should be imperative in nature.
a better one would be `make find command case insensitive`

please take a look at the comments, ack and close the PR.

**Comment by gabrielsim** [comment]
Noted with thanks, and updated!

**Pull request closed without merging**